fe/src/components/ListingTab.vue

```diff
@@ -0,0 +1,76 @@
+<template>
+    <v-app class="pl-10">
+      <v-navigation-drawer>
+        <v-list dense>
+          <v-list-item-group>
+  
+            <!--Listing-->
+            <v-list-item href="#listing" @click="activeOption = 'li'" :class="{'option-active' : activeOption === 'li'}">
+              <v-list-item-content>
+                <v-list-item-title><b>Listing</b></v-list-item-title>
+              </v-list-item-content>
+            </v-list-item>
+  
+            <!--Pricing-->
+            <v-list-item href="#pricing" @click="activeOption = 'pr'" :class="{'option-active' : activeOption === 'pr'}">
+              <v-list-item-content>
+                <v-list-item-title><b>Pricing</b></v-list-item-title>
+              </v-list-item-content>
+            </v-list-item>
+
+            <!--Description-->
+            <v-list-item href="#description" @click="activeOption = 'de'" :class="{'option-active' : activeOption === 'de'}">
+              <v-list-item-content>
+                <v-list-item-title><b>Description</b></v-list-item-title>
+              </v-list-item-content>
+            </v-list-item>
+
+            <!--Photos-->
+            <v-list-item href="#photos" @click="activeOption = 'ph'" :class="{'option-active' : activeOption === 'ph'}">
+              <v-list-item-content>
+                <v-list-item-title><b>Photos</b></v-list-item-title>
+              </v-list-item-content>
+            </v-list-item>
+
+            <!--Amenities-->
+            <v-list-item href="#amenities" @click="activeOption = 'am'" :class="{'option-active' : activeOption === 'am'}">
+              <v-list-item-content>
+                <v-list-item-title><b>Amenities</b></v-list-item-title>
+              </v-list-item-content>
+            </v-list-item>
+
+            <!--Location-->
+            <v-list-item href="#listings" @click="activeOption = 'lo'" :class="{'option-active' : activeOption === 'lo'}">
+              <v-list-item-content>
+                <v-list-item-title><b>Location</b></v-list-item-title>
+              </v-list-item-content>
+            </v-list-item>
+  
+          </v-list-item-group>
+        </v-list>
+
+        <v-divider></v-divider>
+
+        <!--Publish-->
+        <v-list class="d-flex justify-center pt-5 pl-2 pr-2">
+            <v-btn class="flex-grow-1" color="red" rounded="lg" id="btn" w>
+                Publish
```

Comment:
I think this button is not necessary since there will be a save button on the add new listing page
thanks

Reply:
Noted, thank you!